Gopkg.toml

```diff
@@ -132,3 +132,11 @@ required = [
   branch = "master"
   name = "github.com/metal3-io/baremetal-operator"
   source = "https://github.com/openshift/baremetal-operator"
+
+[[constraint]]
+  name = "github.com/ovirt/cluster-api-provider-ovirt"
+  branch = "v1alpha1"
```

Comment:
i'm pretty sure we can't use v1alpha1 APIs for a GA'able platform.

Also since this repo is not controlled in openshift, I would like to see it pinned to a better constraint than just the branch.

Reply:
> i'm pretty sure we can't use v1alpha1 APIs for a GA'able platform.
>

From `Gopkg.toml` 
```
[[constraint]]
  name = "github.com/openshift/cluster-api"
  branch = "openshift-4.2-cluster-api-0.1.0"
```
I see `v1alpha1` is in use. 


> Also since this repo is not controlled in openshift, I would like to see it pinned to a better constraint than just the branch.

There is no release tag. It uses branches per version, similar to what openshift/cluster-api do.

Reply:
> There is no release tag. It uses branches per version, similar to what openshift/cluster-api do.

@abhinavdahiya would you want to pin this to a commit?